conan/cli/commands/new.py

```diff
@@ -24,6 +24,8 @@ def new(conan_api, parser, *args):
     parser.add_argument("-d", "--define", action="append",
                         help="Define a template argument as key=value, e.g., -d name=mypkg")
     parser.add_argument("-f", "--force", action='store_true', help="Overwrite file if it already exists")
+    parser.add_argument("-of", "--output-folder", help="Output folder for the generated files",
```

Comment:
Better use another arg name, this is associated already with the "build" output of ``conan install``, might be a bit confusing